application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -100,6 +103,15 @@ public String getToken() {
         return token;
     }
 
+    /**
+     * Gets the API Key of GitHub to upload pastes via the API.
```

Comment:
A link to the URL or a github tutorial of how to create this token could be helpful.

Think of a new user who wants to use our code and needs to understand how to obtain such a token. Ideally, the Javadoc hints them in the right direction here.